tinyinst.cpp

```diff
@@ -1109,6 +1116,12 @@ void TinyInst::Init(int argc, char **argv) {
   trace_basic_blocks = GetBinaryOption("-trace_basic_blocks", argc, argv, false);
   trace_module_entries = GetBinaryOption("-trace_module_entries", argc, argv, false);
 
+#if defined(ARM64) && defined(__APPLE__)
+  page_extend_modules = GetBinaryOption("-page_extend_modules", argc, argv, true);
+#elif defined(__APPLE__)
+  page_extend_modules = GetBinaryOption("-page_extend_modules", argc, argv, false);

```

Comment:
page_extend_modules will be uninitialized on non-apple builds. It's not currently used on non-apple builds either, but as a general practice would be safer to initialize somewhere before here. Alternately, you could just do 

```
#if defined(ARM64) && defined(__APPLE__)
  page_extend_modules = GetBinaryOption("-page_extend_modules", argc, argv, true);
#else
  page_extend_modules = false
#endif
```

As I don't think there is any reason to have it enabled outside of (Arm64 && Apple) case